spec/autofill-spec.ts

```diff
@@ -12,15 +12,15 @@ describe('autofill', () => {
   it('can be selected via keyboard', async () => {
     const w = new BrowserWindow({ show: true });
     await w.loadFile(path.join(fixturesPath, 'pages', 'datalist.html'));
-    w.webContents.sendInputEvent({ type: 'keyDown', keyCode: 'Tab' });
+    w.webContents.sendInputEvent({ type: 'rawKeyDown', keyCode: 'Tab' });
```

Comment:
Should we maybe document the difference between `rawKeyDown` and `keyDown` in our InputEvent docs? I'm confused about when to use one over the other.

Reply:
This should actually be switched back to `keyDown` now, I updated the code for `sendInputEvent` to convert `keyDown` to `rawKeyDown` under the hood so the behavior doesn't change there.